commands/scanandfixrepos.go

```diff
@@ -33,12 +36,26 @@ func (saf *ScanAndFixRepositories) Run(configAggregator utils.FrogbotConfigAggre
 
 func (saf *ScanAndFixRepositories) scanAndFixSingleRepository(repoConfig *utils.FrogbotRepoConfig, client vcsclient.VcsClient) error {
 	for _, branch := range repoConfig.Branches {
-		err := saf.downloadAndRunScanAndFix(repoConfig, branch, client)
+		shouldScan, checkedCommit, err := saf.shouldScanLatestCommit(context.Background(), repoConfig, client, branch)
+		if err != nil {
+			return err
+		}
+		if !shouldScan {
+			log.Info(fmt.Sprintf("Commit %s has already been scanned by FrogBot", checkedCommit))
```

Comment:
I suggest improving the log message and changing it to the following (needs to be tested):
```
log.Info(fmt.Sprintf("Commit '%s' in repo '%s', branch '%s' has already been scanned. Skipping the scan.", checkedCommit, repoConfig.RepoName, branch))
```